crates/uv/src/commands/tool/run.rs

```diff
@@ -153,12 +153,18 @@ pub(crate) async fn run(
         Err(err) if err.kind() == std::io::ErrorKind::NotFound => {
             match get_entrypoints(&from.name, &environment) {
                 Ok(entrypoints) => {
-                    writeln!(
-                        printer.stdout(),
-                        "The executable `{}` was not found.",
-                        executable.to_string_lossy().cyan(),
-                    )?;
-                    if !entrypoints.is_empty() {
+                    if entrypoints.is_empty() {
+                        writeln!(
+                            printer.stdout(),
+                            "No executables, including `{}`, were found.",
+                            executable.to_string_lossy().cyan(),
+                        )?;
+                    } else {
```

Comment:
I'd leave the above as-is and in the new message say something like "warning: `{from.name}` does not provide any executables". Wdyt?

Reply:
Yeah makes sense, I will push the change in a sec.

Reply:
Done